manifest/resolver.go

```diff
@@ -218,6 +218,23 @@ func (r *Resolver) Sync(l *ui.UI, force bool) error {
 	return nil
 }
 
+// SearchLite returns package references matching the given pattern.
```

Comment:
This comment should mention that `pattern` is a glob, to differentiate it from the regex used in the other method.